runtime/startup_aux.c

```diff
@@ -118,6 +130,31 @@ void caml_parse_ocamlrunparam(void)
         break;
       case 'V': scanmult (opt, &params.verify_heap); break;
       case 'W': scanmult (opt, &caml_runtime_warnings); break;
+      case 'X': {
+        char *name = opt;
+        while (*opt != '\0') {
+          if (*opt == '=') {
```

Comment:
I think this should allow a default of `=1` in the way other options do. At the moment we get:
```console
$ OCAMLRUNPARAM=Xhelp,Xfoo=1 runtime/ocamlrun
Ignored unknown GC tweak 'help,Xfoo'. Use 'Xhelp=1' to list known tweaks
```
where with:
```suggestion
          if (*opt == '=' || *opt == ',') {
```
we get a warning about unknown tweak "foo"